src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/InputJavadocContentLocationFirstLine.java

```diff
@@ -1,7 +1,12 @@
+/*
+JavadocContentLocation
+location = FIRST_LINE
+
+
+*/
+
 package com.puppycrawl.tools.checkstyle.checks.javadoc.javadoccontentlocation;
 
-/** config: location="FIRST_LINE"
- */
 public interface InputJavadocContentLocationFirstLine {
 
     /**
```

Comment:
Slight problem here. The actual violation is on line 12 but if we add text like // violation there, it will no longer be a violation.

Reply:
lets skip it for now

Reply:
We can merge though. It is the case with just one input. Rest of the inputs are fine.